wix-ecom-backend/wix-ecom-backend/guides/1-wix-ecom-backend-intro.md

```diff
@@ -3,27 +3,69 @@ title: Introduction
 ---
 # Introduction
 
-The wix-ecom-backend Velo API allows you to create, manage, and/or retrieve 3 elements of the eCommerce purchase flow: **cart**; **checkout**; **order**.
+The `wix-ecom-backend` Velo APIs are a comprehensive suite of services that address standard and specialized needs in online selling.
```

Comment:
I think the current guidelines  are to refer to "the Wix eCommerce API" in intros rather than the module name.